x-pack/plugins/session_view/common/utils/sort_processes.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I really like the organization of your code base and how things are individualized. Makes it a lot easier to follow!